infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2

```diff
@@ -34,23 +33,17 @@ server {
     add_header X-XSS-Protection "1; mode=block";
 
     location / {
-
+      {{ lobby_proxy_pass }}
       proxy_set_header        Host $host;
       proxy_set_header        X-Real-IP $remote_addr;
       proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
       proxy_set_header        X-Forwarded-Proto $scheme;
-
-      # Fix the “It appears that your reverse proxy set up is broken" error.
-      proxy_pass          http://localhost:8080;
       proxy_read_timeout  90;
-
-      proxy_redirect      https://localhost:443 https://localhost:8080;
```

Comment:
This 'proxy_redirect' line is seemingly unneeded. While testing, this line did not seem to matter.